tornado/locks.py

```diff
@@ -0,0 +1,62 @@
+__all__ = ['Condition']
```

Comment:
Yes. Copy it from another file and make it `Copyright 2015 The Tornado Authors`. Also copy the `__future__` import just so everything is consistent.